### PR TITLE
Prepare for v2.3.4 release

### PR DESCRIPTION
**Pull request by vigh-m**
**Description of changes:**
This PR prepares for the v2.3.4 release. Changes include:
- `CHANGELOG.md` update for the v2.3.4 release.
- Bumping `release-version` to `2.3.4` in Twoliter

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by ytsssun — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# v2.3.4 (2024-08-19)
+
+## Build Changes
+* sundog: strip "settings" prefix from populated settings ([#94])
```
Comment (yeazelm) [inline CHANGELOG.md]:
```suggestion
* Fix a bug in `sundog` where settings would be rendered incorrectly from the "settings" prefix ([#94])
```

**Comment by vigh-m** [comment]
Updated wording on the changelog entry

File: CHANGELOG.md (hunk 1 above)
Comment (bcressey) [reply CHANGELOG.md]: > Fix a bug in `sundog` where settings would be rendered incorrectly from the "settings" prefix

It wasn't a rendering bug. It was more that it couldn't detect that settings were populated. But a higher-level description might be better:
> Fix a bug in `sundog` that caused it to regenerate populated settings

**Review by ytsssun — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,22 @@
+# v2.3.4 (2024-08-19)
+## OS Changes
+
+* Update libnvidia-container to v550.54.14 and nvidia-container-toolkit to v1.16.1 ([#88])
+
+## Settings API Changes
```
Comment (bcressey) [inline CHANGELOG.md]: Have we used this category before? The settings API is part of the OS, so I would just include it in "OS Changes".

**Review by ytsssun — APPROVED** [review]

**Review by jpculp — APPROVED** [review]

**Comment by vigh-m** [comment]
Update moving changlog entries under the right headings with the updated wording

**Review by yeazelm — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**